dev-python/python-msrplib/python-msrplib-0.19.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="Client library for MSRP protocol and its relay extension"
+HOMEPAGE="http://sipsimpleclient.org"
+SRC_URI="https://github.com/AGProjects/${PN}/archive/release-${PV}.tar.gz"
```

Comment:
Please use `->` to rename the file to `${PN}-release-${PV}.tar.gz`. Having a distfile named `release-*` is not very readable, and can collide with other packages.